docs/api/service.md

```diff
@@ -38,12 +38,12 @@ The event and its data must be defined in the [service's definition file](../gui
 #### EmitEventRequest
 The request's data for the `EmitEvent` API.
 
-**Example:**
+Example:
 ```json
 {
-  "token":     "__SERVICE_TOKEN_FROM_ENV__",
-  "eventKey":  "__EVENT_KEY__",
-  "eventData": "{\"foo\":\"hello\",\"bar\":false}"
+"token":     "__SERVICE_TOKEN_FROM_ENV__",
+"eventKey":  "__EVENT_KEY__",
+"eventData": "{\"foo\":\"hello\",\"bar\":false}"
```

Comment:
@antho1404 I generate the doc for api and the template seems a bit different.
It remove the space at the beginning of the lines and the bold around example.
Do you know why?

Reply:
try updating the branch ilker fixed that on the PR https://github.com/mesg-foundation/core/pull/541#discussion_r234115529

Reply:
thanks!